content/doc/book/architecting-for-manageability.adoc

```diff
@@ -167,38 +159,11 @@ docker run -d --dns=172.17.42.1 --name joc-1 --volumes-from storage -e JENKINS_H
 
 Note that Docker only supports one mounted volume at a time, so if you are planning on running multiple test instances on Docker, all of their _$JENKINS_HOME_s will need to be version controlled in the same GitHub repo.
```

Comment:
I this is not actually true anymore.  In fact some of the [docker run documentation](https://docs.docker.com/engine/reference/commandline/run/) even references mounting multiple volumes at once on the run command